doc/command_line_tool.md

```diff
@@ -38,15 +38,27 @@ In order to build the grpc command line tool from a fresh clone of the grpc
 repository, you need to run the following command to update submodules:
 
 ```
-git submodule update --init
+$ git submodule update --init
 ```
-
+---
```

Comment:
Please remove the `---` and the `Unix user`/`OSX user` section headings. We can include the command-line for homebrew without breaking this into separate sections.

Reply:
done